transport/src/test/java/io/netty/bootstrap/BootstrapTest.java

```diff
@@ -313,6 +314,22 @@ public void run() {
                         }
                     });
                 }
+
+                @Override
+                protected void doResolveAll(
+                        final SocketAddress unresolvedAddress, final Promise<List<SocketAddress>> promise)
+                        throws Exception {
+                    executor().execute(new Runnable() {
+                        @Override
+                        public void run() {
+                            if (success) {
+                                promise.setSuccess(Collections.singletonList(unresolvedAddress));
+                            } else {
+                                promise.setFailure(new UnknownHostException());
```

Comment:
I think we either should include the "unresolvedAddress" in the exception message or just reuse a static instance of UnknownHostException.